Assets/MixedRealityToolkit.Services/InputSystem/Utilities/ScaleMeshEffect.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Input.Utilities
```

Comment:
Should this be in Services, or SDK? @davidkline-ms I often wonder about this, whether to put things in services or SDK. What are the rules to follow for deciding if something should go into services or SDK? Maybe a one sentence rule of thumb to follow? Here's a strawman: "If it related to getting data, it goes into services. Otherwise (UI, interactions, etc) put it in SDK"

Reply:
To be honest I don't understand the divide between SDK, Services, and Providers. What does the dependency graph look like? And, can they live on their own? I'm happy to put this wherever makes sense. :)

Reply:
You can figure this out by looking at the asmdef files. It looks like SDK depends on a bunch of stuff, but services usually depends only on MixedRealityToolkit.

I think if it isn't a core MRTK service like the input system, spatial mesh system, or boundary system, it should go into SDK.

Reply:
Sounds good! I'll move this into SDK.

Reply:
We spoke offline, going to keep this script in Services because MixedRealityCanvasInspector depends on it.